frontend/components/CustomLink/_styles.scss

```
@@ -46,8 +29,32 @@
+    color: $core-fleet-green;
+    text-decoration-color: $core-fleet-green;
+
+    &:hover {
+      color: $core-fleet-green-over;
+      text-decoration-color: $core-fleet-green-over;
+    }
+
+    &:focus-visible {
+      color: $core-fleet-green-over;
+      text-decoration-color: $core-fleet-green-over;
+      text-decoration: none;
```

Stylelint (17.14.0)

[error] Overridden property "text-decoration-color" by shorthand "text-decoration" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)